devito/types/parallel.py

```diff
@@ -44,8 +44,10 @@ class NThreadsBase(NThreadsAbstract):
 
     @cached_property
     def default_value(self):
-        return int(os.environ.get('OMP_NUM_THREADS',
-                                  configuration['platform'].cores_physical))
+        return int(os.environ.get(
+            'OMP_NUM_THREADS',
+            configuration['platform'].cores_physical_per_numa_domain
```

Comment:
Since you touched this..... need a `// mpi_ranks`

Reply:
why? this is already physical cores "per numa domain"

I could take `.core_physical` and divide it by `mpi_ranks`, which might be slightly better than `cores_physical_per_numa_domain`, for some exotic use cases...